tests/nightly/unity/task.yaml

```diff
@@ -5,19 +5,14 @@ environment:
 
 systems: [ubuntu-16.04-64]
 
-execute:
-    # FIXME Test is broken. Downloads HUNDREDS and HUNDREDS of packages,
-    # and asks for input on configuration file diffs.
```

Comment:
The diff can be done-away with `DEBCONF_FRONTEND=noninteractive` or something like this (I don't recall precisely)